src/lifecycle/events.js

```diff
@@ -28,7 +28,7 @@ function makeDelegateHandler (elem, handler, parsed) {
       if (matches(current, selector)) {
         readonly(e, 'currentTarget', current);
         readonly(e, 'delegateTarget', elem);
-        return handler(e);
+        return handler(elem, e);
```

Comment:
What about the rest of the Readme file then? Lots of `elem` instead of `element` everywhere there.